sound/soc/codecs/rt715.c

```diff
@@ -737,7 +737,19 @@ static int rt715_set_bias_level(struct snd_soc_component *component,
 	return 0;
 }
 
+static int rt715_probe(struct snd_soc_component *component)
+{
+	int ret;
+
+	ret = pm_runtime_resume(component->dev);
```

Comment:
@plbossart we shoudl consider introducing a macro for this runtime resume omitting -EACCES